cmd/client-observer/main.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/drand/drand/key"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/urfave/cli/v2"
+)
+
+var urlFlag = &cli.StringFlag{
+	Name:  "url",
+	Usage: "Root URL for fetching randomness.",
+}
+
+var groupKeyFlag = &cli.StringFlag{
+	Name:  "group-key",
+	Usage: "Path to TOML file containing the group key.",
+}
+
+var metricsFlag = &cli.StringFlag{
```

Comment:
you can set a `Value` key on this, and then always get a value with that as a default